frontend/src/features/admin-form/settings/components/EmailFormSection.tsx

```diff
@@ -48,22 +79,47 @@ export const EmailFormSection = ({
     [initialEmailSet, mutateFormEmails],
   )
 
-  useEffect(() => reset({ emails: initialEmails }), [initialEmails, reset])
+  useEffect(() => reset({ emails: settings.emails }), [settings.emails, reset])
+
+  const emailModeDescription = `Add at least **2 recipients** to prevent loss of response.`
+  const storageModeDescription = `FormSG securely stores responses in an encrypted format and does not retain any associated emails.`
 
   return (
     <>
-      <MRFAdvertisingInfobox />
+      {isFormPublic ? (
```

Comment:
Would recommend refactoring these ternary into a separate component. Usage of multiple ternaries is a sign of code smell - readability.

Something along the lines of
```
<EmailHeader
  isFormPublic={isFormPublic}
  isEmailInputDisabled={isEmailInputDisabled}
  isFormResponseModeEncrypt={...}
/>

const EmailHeader = ({isFormPublic, isEmailInputDisabled, responseMode) => {
  if(isFormPublic) return ...
  if(isEmailInputDisabled) return ...
  if(isFormResponseModeEncrypt) return ...
  return null
}
```

Reply:
The render section of a React Component should be kept as clean as possible. Clean here refers to how close it looks like HTML (presentation).

The business logic should be minimized as much as possible in the presentation section.

Reply:
Thanks for the advice! This is much cleaner, got to get a hang of writing good frontend code 😅

Will take note of the point on code smell as well.